python/doc/usecases/use_case_stiffened_panel.rst

```diff
@@ -0,0 +1,207 @@
+
+- :math:`t_s` is the thickness of the panel main surface;
+
+- :math:`E_s` is the Young modulus of the panel main surface;
+
+- :math:`\nu_s` is the Poisson coefficient of the panel main surface;
+
+- :math:`D=\frac{E_st_s^3}{12(1-\nu_s^2)}` is the bending coefficient of the
+  panel main surface;
+
+- :math:`k_{xy}` is the load factor associated to shear buckling. It is given as
+  a function of :math:`\frac{b_0}{a}` through the empirical relation
+  :math:`k_{xy}=5.35 + 4\left(\frac{b_0}{a}\right)^2`
+
+It is more convenient to use the shear force :math:`N_{xy}` instead of the shear
+stress component :math:`\tau_{xy}`. It leads to the relation:
+
+.. math::
+   N_{xy}=q_1+q_c
+
+where :math:`q_1` abd :math:`q_c` are the shear fluxes in the panel main surface
+and its stiffener. They are given by:
+
+.. math::
+   q_1=\tau_{xy}t_s=2G_sh_0t_s\frac{\partial^2w}{\partial x\partial y}
+
+and
+
+.. math::
+  q_c=\frac{G_ct_cp}{\ell}\left[h-2h_0+\frac{h_c}{2p}(f_1-f_2)\right]\frac{\partial^2w}{\partial x\partial y}
+
+where
+
+- :math:`G_s=\frac{E_s}{2(1+\nu_s)}` is the shear modulus of the panel main
+  surface;
+
+- :math:`\frac{\partial^2w}{\partial x\partial y}` is the torsion strain of the
+  panel main surface;
+
+- :math:`G_c=\frac{E_c}{2(1+\nu_c)}` is the shear coefficient of the stiffener;
+
+- :math:`t_c` is the thickness of the stiffener;
+
+- :math:`h_c` is the distance between the mean surfaces of the stiffener hat and
+  foot;
+
+- :math:`h=h_c+\frac{t_c+t_s}{2}` is the distance between the mean surfaces of
+  the stiffener hat and the panel main surface;
+
+- :math:`f_1` is the width of the foot of the stiffener;
+
+- :math:`f_2` is the width of the hat of the stiffener;
+
+- :math:`p` is the half-widht of the stiffener;
+
+- :math:`R` is the radius of the circular part of the stiffener;
+
+- :math:`\theta` is the angle of the circular part of the stiffener;
+
+- :math:`\ell` is the length of the stiffener flank;
+
+- :math:`d=\frac{\ell-f_2}{2}-R\theta- :math:` is the half-lenght of the straight
+  part of the flank of the stiffener;
+
+- :math:`A=\ell t_c` is the area of the section of an half-ondulation;
+
+- :math:`\bar{A}=A+pt_s+\frac{1}{2}(f_1-f_2)t_c` is the area of the section of
+  the full panel (main surface and stiffener) bounded by :math:`p`;
+
+- :math:`h_0=\frac{1}{2\bar{A}}\left[A(h_c+t_c+t_s)+\frac{1}{2}t_c(f_1-f_2)(t_c+t_s)\right]`
+  is the distance between the mean surface of the panel main surface and the
+  global geometric center of the pa,nel;
+
+It leads to:
+
+.. math::
+  N_{xy}=q_1(1+q_c/q1)=\tau_{xy}t_s\left[1+\frac{1}{4}\frac{G_ct_c}{G_st_s}\frac{\left(2p(h-2h_0)-h_c(f_1-f_2)\right)}{h_0\ell}\right]
+
+and finally, :math:`(N_{xy})_{cr}` is given by:
+
+.. math::
+  (N_{xy})_{cr}=\bigg[5.35 + 4\left(\frac{b_0}{a}\right)^2\bigg]\bigg[\frac{\pi^2}{b_0^2}\frac{E_st_s^3}{12(1-\nu_s^2)}\bigg]\bigg[1+\frac{1}{4}\frac{G_ct_c}{G_st_s}\frac{(2p(h-2h_0)-h_c(f_1-f_2))}{h_0\ell}\bigg]
```

Comment:
```suggestion
  (N_{xy})_{cr} = \left(5.35 + 4\left(\frac{b_0}{a}\right)^2\right) \left(\frac{\pi^2}{b_0^2}\frac{E_st_s^3}{12(1-\nu_s^2)}\right) \left(1+\frac{1}{4}\frac{G_ct_c}{G_st_s}\frac{(2p(h-2h_0)-h_c(f_1-f_2))}{h_0\ell}\right)
```

Reply:
I will keep \bigg and switch to parentheses, because I find it ugly when two blocks have delimiters of very different sizes.

Reply:
The word "ugly" seems a little to much for this equation. Here is the output:

![image](https://github.com/openturns/openturns/assets/31351465/04175e0e-5343-4a3b-be3d-1195262e5657)

Notice the excess of parentheses in the third parenthesis, in the numerator of the fraction.

Reply:
At this stage, I think that you are confusing two different goals in the presentation.
- You derive the equation for $N_{xy}$ and substitute for $k_{xy}$ and $D$, because that it is needed for alleviating a limitation of Exprtk that cannot handle exact derivation when there is a complicated evaluation tree.
- It is not necessary to get the full expanded expression to fully understand the model.

In the end, the expression:

![image](https://github.com/openturns/openturns/assets/31351465/aeab2b33-e9e8-4d99-b376-031a77030660)

is the simplest expression that describes the model isn'it?

Reply:
\bigg is a relief of a first version of the equations, where the front factor was expessed in terms of $r=\frac{b_0}{a}$. Now it is useless, well pointed!